### ISSUE TITLE
Skipping task ':nativeImage' as it is up-to-date (even on a clean build...)

### ISSUE THREAD

**Issue by remkop** [mike-neck/graalvm-native-image-plugin#12]
I am seeing the below output now when I try to build [my project](https://github.com/remkop/picocli-native-image-demo) locally, either on Windows or on Ubuntu:

```
> Task :nativeImage UP-TO-DATE
Skipping task ':nativeImage' as it is up-to-date.
:nativeImage (Thread[Execution worker for ':',5,main]) completed. Took 0.0 secs.
```

This is weird because I am doing a `.gradlew clean build` and there are no existing binaries from a previous build. 

As a result, the build does not produce any native image files any more...

Any ideas why this could be happening?

**Comment by mike-neck** [comment]
Sorry for inconvenient.

Calculation for `UP-TO-DATE` has something wrong. For the immediate workaround for this behaviour, please configure `nativeImage` task as follows.

```groovy
tasks.nativeImage.outputs.upToDateWhen { false }
```

**Comment by mike-neck** [comment]
Fixed issue via #13 . This change will be released as version `0.2.1`.

**Comment by remkop** [comment]
Great, looking forward to it!
